src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java

```diff
@@ -122,6 +124,15 @@ public void setInstanceCap(int instanceCap) {
         this.instanceCap = instanceCap;
     }
 
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    @DataBoundSetter
+    public void setIdleMinutes(int instanceCap) {
+        this.idleMinutes = idleMinutes;
```

Comment:
@zoumo: This is why the build fails. I guess you need to change the method argument to `idleMinutes`.